daemon/daemon_unix.go

```diff
@@ -452,7 +452,7 @@ func verifyPlatformContainerResources(resources *containertypes.Resources, sysIn
 		resources.KernelMemory = 0
 	}
 	if resources.KernelMemory > 0 && resources.KernelMemory < linuxMinMemory {
-		return warnings, fmt.Errorf("Minimum kernel memory limit allowed is 4MB")
+		return warnings, fmt.Errorf("Minimum kernel memory limit allowed is 6MB")
```

Comment:
I'm having a touch of deja-vu: https://github.com/moby/moby/pull/43214#discussion_r801173149 :eyes: :sweat_smile:

Reply:
hehe, yes, that's where these commits came from, but thought to move them out of that PR, as they already apply to the current API (and perhaps making the other PR slightly easier / smaller to review)